lib/mapping/CMap.h

```diff
@@ -424,6 +424,7 @@ class DLL_LINKAGE CMap : public CMapHeader
 	void serialize(Handler &h, const int formatVersion)
 	{
 		h & static_cast<CMapHeader&>(*this);
+		h & triggeredEvents; //from CMapHeader

```

Comment:
Should we increase minimum serialization version?

Reply:
We already do it for 1.1
I don't think we need to increase each build where we break compatibility.